### DISCUSSION TITLE
Search, Search, Search

### DISCUSSION THREAD

**Discussion by brucegomes** [IntelliTect/EssentialCSharp.Web#12] [Feedback and Ideas] [answered]
It would be nice to be able to search for a subject, topic, or content throughout the whole book. 

User Case: Let's say I want to see everything related to threads in the whole book, I'd like to type "Threads" and the side panel to show me all topics, subjects, pages where threads are talked about. If this search could be filtered by beginner/advanced like the book is, it would be even better.

**Comment by BenjaminMichaelis** [comment]
Thanks for the feedback @brucegomes! The first iteration of search is live, with more adjustments coming soon!

**Reply by BenjaminMichaelis** [reply, marked as answer]
Resolved in https://github.com/IntelliTect/EssentialCSharp.Web/pull/585 !
